src/swap.ts

```diff
@@ -153,7 +155,7 @@ async function compareMessagesAndTransaction(
   msgAccept?: proto.SwapAccept
 ): Promise<void> {
   // decode the transaction.
-  const decodedFromRequest = decodePsbt(msgRequest.getTransaction());
+  const decodedFromRequest = decodePsbt(msgRequest.transaction);
```

Comment:
better to use getters when retrieveing data from a proto response, can you please revert such changes?